llvm/test/Transforms/MergeFunc/mergefunc-preserve-vfe-intrinsics.ll

```diff
@@ -0,0 +1,34 @@
+; This test contains three identical functions, aside from the metadata
+; they pass to a function call. This test verifies that the function merger
+; pass is able to merge the two functions that are truly identifical,
+; but the third that passes different metadata is preserved
+
+; RUN: opt -passes=mergefunc -S %s | FileCheck %s
```

Comment:
Use update_test_checks.py please, and move the test description below the RUN line.